templates/typescript/samples/sample-assistant/src/index.ts

```diff
@@ -77,12 +77,16 @@ const credentialProvider: SimpleCredentialProvider = new SimpleCredentialProvide
 // Register the skills configuration class.
 const skillsConfig: SkillsConfiguration = new SkillsConfiguration(appsettings.botFrameworkSkills as IEnhancedBotFrameworkSkill[], appsettings.skillHostEndpoint);
 
+let authenticationConfiguration = new AuthenticationConfiguration();
+
 // Register AuthConfiguration to enable custom claim validation.
-const allowedCallers: string[] = [...skillsConfig.skills.values()].map(skill => skill.appId);
-const authenticationConfiguration = new AuthenticationConfiguration(
-    undefined,
-    allowedCallersClaimsValidator(allowedCallers)
-);
+if(skillsConfig.skills.size > 0){
+    const allowedCallers: string[] = [...skillsConfig.skills.values()].map(skill => skill.appId);
+    const authenticationConfiguration = new AuthenticationConfiguration(
```

Comment:
```suggestion
    authenticationConfiguration = new AuthenticationConfiguration(
```

If you have defined this variable as `let` you can override it. If not, this `const` won't be used outside